tests/webview/integration/test_rex_redirects.py

```diff
@@ -31,28 +30,24 @@ def test_archive_is_still_reachable(archive_base_url, rex_base_url):
 @markers.rex
 @markers.nondestructive
 def test_redirecting_to_rex_from_within_webview(
-    webview_base_url, rex_base_url, selenium, redirecting_books_titles
+    webview_base_url, rex_base_url, selenium, rex_released_books
 ):
     """Webview needs to redirect to REX when any of the featured books is a REX book.
     https://github.com/openstax/cnx/issues/401
     """
 
-    for eligible_book in redirecting_books_titles:
-        # GIVEN the home page
-        home = Home(selenium, webview_base_url).open()
+    for book in rex_released_books:
 
-        # WHEN we click on a REX book
-        for book in home.featured_books.openstax_list:
+        # GIVEN the cnx url of a redirecting book
+        cnx_book_url = f"{webview_base_url}/contents/{book}"
```

Comment:
I'm not sure this is a good test. You are building the url yourself and then making a request. I believe the intent of the test is to click the link from the webview homepage no?